bapt/cli.py

```diff
@@ -24,16 +24,29 @@ def main():
     parser.add_argument('-f', '--filename', default=None,
                         help='Path to file containing alignment information.')
     parser.add_argument('-n', '--name',
-                        help='List of compound names (comma seperated).' +
-                        'Must be used in conguction with --ip and --ea.')
+                        help='List of compound names (comma seperated). ' +
+                        'Must be used in conguction with --ip and --ea. ' +
+                        'Use $_x$ and $^y$ for subscript and superscript.')
     parser.add_argument('-i', '--ip',
                         help='List of ionisation potentials (comma separated).')
     parser.add_argument('-e', '--ea',
                         help='List of electron affinities (comma separated).')
+    parser.add_argument('-cbo', '--cbo', default=None,
+                        help='List of conduction band offsets (comma separated). ' +
+                        '(Relative to first compound)(-> No vacuum alignment)')
+    parser.add_argument('-vbo', '--vbo', default=None,
+                        help='List of valence band offsets (comma separated). ' +
+                        '(Relative to first compound)(-> No vacuum alignment)')
+    parser.add_argument('-eg', '--eg',
```

Comment:
I think `--band-gap`  is actually a better long argument. It is more clear what it means immediately. You can use `-b` as an abbreviation.

Reply:
Sounds good, will do.